### PR TITLE
issue#17 create navbar sub-menu

### LINKED ISSUE #17: Navigation to sub menus
## Description
As a user,
I want to see the BIT public landing page/ Homepage,
so that I can see programs I like after login.

## Mocks
See the latest mock under https://github.com/anitab-org/bridge-in-tech-web/issues/4

### Update [Required]
- [ ] About BridgeInTech section
- [ ] Benefits BIT section
- [ ] Recent mentoring opportunities section
- [ ] Contact form section (with details)

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
2 - 3 days

### PR DESCRIPTION
**Pull request by mtreacy002**
### Description
Add sub-menu so user can navigate to member's private pages under `My Space`

Fixes #17 

### Type of Change:
- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Checked that all functionalities that have been done so far work as expected.

### Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by mtreacy002** [comment]
Update @anitab-org/bridgeintech-maintainers and @meenakshi-dhanani. I've ecompleted the Navbar sub-menu tasks. Can you please reeview? In the meantime I'll work on the View /user/personal_details page. Thanks

File: src/Routes.js [hunk 1]
```diff
@@ -30,14 +31,19 @@ export default function Routes() {
             <Login />
           </Route>
           <ProtectedRoute
-            exact
-            path="/my-space"
-          >
-            <MySpace />
+            exact path="/my-portfolio">
```
Comment (meenakshi-dhanani) [inline src/Routes.js]: can't we just say /portfolio /profile ? this my my doesn't sound right

File: src/myspace/MyPortfolio.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+export default function MyPortfolio() {
```
Comment (meenakshi-dhanani) [inline src/myspace/MyPortfolio.jsx]: Again I just find My My in everything unnecessary.

File: src/Routes.js (hunk 1 above)
Comment (mtreacy002) [reply src/Routes.js]: Updated. I've removed `My`. Now it's only `Portfolio/Profile`
Screen Shot 2020-07-13 at 1 03 16 am

File: src/myspace/MyPortfolio.jsx (hunk 2 above)
Comment (mtreacy002) [reply src/myspace/MyPortfolio.jsx]: done. now without `My`

**Review by foongminwong — DISMISSED** [review]
Here's a test, looks good to me:
test-bit-pr-37

File: src/Navigation.js [hunk 3]
```diff
@@ -1,44 +1,122 @@
...
+            <Navbar.Brand href="/">
+                <img
+                    className="d-inline-block align-top"
+                    src="public/assets/images/AnitaBLogo.png"
+                    alt="Logo"
+                />
+            </Navbar.Brand>
+            <Navbar.Toggle aria-controls="basic-navbar-nav" />
+            <Navbar.Collapse id="basic-navbar-nav">
+
```
Comment (meenakshi-dhanani) [inline src/Navigation.js]: why so many lines?

File: src/Navigation.js [hunk 4]
```diff
@@ -1,44 +1,122 @@
...
+                                                <Nav.Link as={Link} to="/about">About</Nav.Link>
+                                                <Nav.Link as={Link} to="/benefit">Benefit</Nav.Link>
+                                                <Nav.Link as={Link} to="/contact">Contact</Nav.Link>
+                                            </Nav>
+
+                                        </Accordion.Collapse>
+                                    </Card>
+                                </Accordion>
+                                </Nav.Item>
+
```
Comment (meenakshi-dhanani) [inline src/Navigation.js]: why so many lines

File: src/Navigation.js [hunk 5]
```diff
@@ -1,44 +1,122 @@
...
+                                        <Card.Header>
+                                            <Accordion.Toggle as={Link} to="my-portfolio" eventKey="0">My Space</Accordion.Toggle>
+                                        </Card.Header>
+                                        <Accordion.Collapse eventKey="0">
+                                            <Nav>
+                                                <Nav.Link as={Link} to="/portfolio">Portfolio</Nav.Link>
+                                                <Nav.Link as={Link} to="/profile">Profile</Nav.Link>
+                                                <Nav.Link as={Link} to="/request-history">Request History</Nav.Link>
+                                            </Nav>
+
```
Comment (meenakshi-dhanani) [inline src/Navigation.js]: there is a closing tag and indentation. why so many lines?

File: src/Navigation.js [hunk 6]
```diff
@@ -1,44 +1,122 @@
...
+                                            <Accordion.Toggle as={Link} to="/members" eventKey="0">Members</Accordion.Toggle>
+                                        </Card.Header>
+                                    </Card>
+                                </Accordion>
+                            </Nav.Item>
+                            <Nav.Item>
+                            <Accordion>
+                                    <Card>
+                                        <Card.Header>
+                                            <Accordion.Toggle as={Link} to="my-portfolio" eventKey="0">My Space</Accordion.Toggle>
```
Comment (meenakshi-dhanani) [inline src/Navigation.js]: are there two routes now? /portfolio and /my-portfolio?

Comment (mtreacy002) [reply src/Navigation.js]: nop. that's a bug. I've fixed it now. Thanks for pointing that out, @meenakshi-dhanani 😁

File: src/Navigation.js (hunk 5 above)
Comment (mtreacy002) [reply src/Navigation.js]: as above 😁

File: src/Navigation.js (hunk 3 above)
Comment (mtreacy002) [reply src/Navigation.js]: sorry, I missed that. It's fixed now.

File: src/Navigation.js (hunk 4 above)
Comment (mtreacy002) [reply src/Navigation.js]: as above 😁

**Comment by mtreacy002** [comment]
Update @meenakshi-dhanani . I've just pushed the requested changes. Can you please re-review? Thanks

**Review by meenakshi-dhanani — APPROVED** [review]

**Comment by meenakshi-dhanani** [comment]
@foongminwong can you approve again?

**Review by foongminwong — APPROVED** [review]

**Pull request merged**